modules/javafx.web/src/main/java/com/sun/javafx/webkit/prism/WCGraphicsPrismContext.java

```diff
@@ -502,6 +507,51 @@ protected boolean shouldCalculateIntersection() {
         return false;
     }
 
+    private void fillOrDrawShapeMasked(Graphics g, Shape shape, Paint paint, BasicStroke stroke, boolean isFill) {
+
+        FilterContext filterContext = getFilterContext(g);
+        Rectangle clipRectangle = getClipRectNoClone();
+
+        PrDrawable imagePrDrawable = (PrDrawable)Effect.getCompatibleImage(filterContext,
+                g.getRenderTarget().getContentWidth(), g.getRenderTarget().getContentHeight());
+        Graphics g1 = imagePrDrawable.createGraphics();
+        state.apply(g1);
+        g1.setPaint(paint);
+        if(stroke != null) {
```

Comment:
Minor: there should be a space after the `if`.

Reply:
Fixed